### DISCUSSION TITLE
Simon's build

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#47] [Show and tell]
![20230818_223917](https://github.com/dmt-labs/modular-biped/assets/34891324/145d3e37-6b8b-4b57-a622-e75d3673132a)

The foot, remodelled to use M3x30 screws, and replace the spacers. I imported Dans STL's to ensure I replicated the geometry.
There are still a few adjustments to make as the servo was very tight. I also have to re commission printer number 2 as I'm currently using a printer that uses 2.85mm and Amazon only seems to sell PLA+ in 1.75mm. Alternatively I could replace the Wade extruder with another. Decisions decisions…..Unfortunately Printer 2’s RAMPs board and Arduino Mega were damaged when one of my psu’s died…..

**Comment by WhoIsMrSentry** [comment]
It looks nice, but I have a suggestion for you, move the height axis of the printer you are printing to a little closer to the table, you can make the base (where the tip of the servo motor is) better, now it looks like a pla line.

**Reply by Chompalot** [reply]
I have to manually set the height of the extruder above the bed, it's an old RepRap without any auto bed leveling etc. I'm waiting for the PLA+ to arrive before I continue. I'm not sure I can set it any closer.

**Comment by danic85** [comment]
Thanks for sharing on the community, it's great to see how people are using the files. If you need help with anything feel free to reach out.

👏

**Comment by Chompalot** [comment]
Progress so far, I can see issues with the gear alignment coming up. I'll have to dabble with Blender to mod the stl's  of the servo gear
![20230825_131436](https://github.com/dmt-labs/modular-biped/assets/34891324/64545546-789a-471c-9520-0b7cf8ce8565)

**Comment by Chompalot** [comment]
![20230905_112015](https://github.com/dmt-labs/modular-biped/assets/34891324/64329223-3dc2-4f46-853e-fe9c125b6305)
The slowly growing collection of parts. I've also had a lot of fun with Blender making slight adjustments, and even adapting the camera mount for a standed Pi camera (V2)

**Reply by danic85** [reply]
Nice work!

**Comment by Chompalot** [comment]
Progress on the build is slow but steady. Not a good time for my soldering iron to break though. Fortunately Amazon has come to the rescue :)

Next I need to centre the servo's then pin the servo horns to the gears.

![20230916_181739](https://github.com/dmt-labs/modular-biped/assets/34891324/a4bea1be-8602-4d4e-896a-5ce6088fcbe0)

I was wondering how dificult it would be to swap out the boards in the servo, to change them from pulse with controlled to a serial data control. It would certainly simplify the wiring if it were possible.

The I found this https://www.sparkfun.com/products/retired/9014

maybe a future improvement?

**Reply by Chompalot** [reply]
Also https://github.com/SUSF-Robotics-and-Software/OpenServo

**Comment by danic85** [comment]
Nice work! I always wondered what it would look like in black. 😂

Yeah serialising the servo connectors would be great. If you manage to get it working let me know because I'd definitely do that on my side too.

Any problems during the build that I can update on the docs or files to help people?


________________________________

**Comment by Chompalot** [comment]
Its hard to say about problems as I'm mostly remodelling parts as I go, so I dont have to use spacers, just M3x30 and M3x40.
A recess in the gear for the servo horn (although its a disc) so the gear line up. Also my printer had trouble with the cradle assembly that joins the legs to the body, so I broke it into pieces that are bolted together

![20230916_191017](https://github.com/dmt-labs/modular-biped/assets/34891324/ef6bed41-0db5-4e88-8e11-f38ac9d34bf2)

The body stl has some extra bits in,  blank plates are these required or were these to blank holes?

![body-split](https://github.com/dmt-labs/modular-biped/assets/34891324/99fab313-2b9e-4de0-9542-a0bd58b2461b)

I actually printed my own spacers to join the body together :)
I also printed axles for the bearings in the legs, rather than using M8 machine screws, firstly as I dont have the right size to hand and also to reduce the overall weight. steel being 7900kg/cubic metre or 7.9g/cc

**Reply by danic85** [reply]
Yes some of the holes in the plates were to allow other mounting points for different hardware - they're not all used.
I printed the cradle at 90 degrees with supports (so front facing down), but I might have just been lucky.

**Comment by WhoIsMrSentry** [comment]
Hello @Chompalot , you have come a long way. It looks very nice. Do you have an stl file of all the parts you used instead of standing screws? I used cable ties. This would be more durable.

**Reply by Chompalot** [reply]
Hi Sentry, I do have all the stl's but it'd take a while to go through them to get the right versions.
I'm not sure about cable ties as I'd be concerned about rigidity.
PLA plus does seem to be far better than standard PLA mechanically speaking, the bond between the layers seems to be far better.

**Comment by Chompalot** [comment]
Progress - power in, currently 12V PSU as that's what I have to hand, throught he power selector switch then to a piece of Vero board , to act as a distribution area. An XL4005 converter for the servo's and a smaller converter for the Arduino. I was going to use 2xXL4005's but had a little accident and so I've had to order another :)
![power-01](https://github.com/dmt-labs/modular-biped/assets/34891324/55eb464b-5e6c-41ce-9f26-2777009ecc2b)

For the Servo control side - Arduino Nano and some 90 degree headers

![arduino-01](https://github.com/dmt-labs/modular-biped/assets/34891324/a99fd897-162b-4751-9763-5084f8eef2d5)

just modelling a plate for mounting these parts.

**Comment by Chompalot** [comment]
A pcb seems like a great idea right about now, however some progress, with a mounting plate and better routing for the wire between plate so they don’t have to be disconnected when disassembling. I thought I'd ordered the gyro, but apparently not so waiting for that to arrive, but a space has been made for it to be mounted.
The vero board is for the voltage divider and connections for SCL, SDA, and power between the nano gyro and Pi

![20231016_112923nano1](https://github.com/dmt-labs/modular-biped/assets/34891324/75062a79-f4fb-4092-b531-9eaba734fa01)

**Comment by Chompalot** [comment]
Some progress sort of
https://youtube.com/shorts/3lKJ5vnFhLs?si=-ggv2Ok0eEsVrlNF
